src/app-layout/__tests__/global-breadcrumbs.test.tsx

```diff
@@ -60,7 +60,8 @@ afterEach(() => {
 
 describeEachAppLayout({ themes: ['refresh-toolbar'], sizes: ['desktop'] }, () => {
   test('renders normal breadcrumbs when no app layout is present', async () => {
-    await renderAsync(<BreadcrumbGroup items={defaultBreadcrumbs} />);
+    render(<BreadcrumbGroup items={defaultBreadcrumbs} />);
+    await delay();
```

Comment:
For the cases where we test no-deduplication mode we keep the delay, because there is no that clue

Reply:
The test "when multiple nested app layouts rendered, the outer instance receives breadcrumbs" is still flaky.

Reply:
And also this one: "when multiple breadcrumbs instances are present the latest is applied"

Reply:
And this: "renders breadcrumbs adjacent to app layout"

Reply:
How do you see that?

Reply:
I run all tests in this file a few hundred times.

Reply:
Where can I see this build?

Reply:
Here: https://github.com/cloudscape-design/components/pull/2677

Reply:
Fixed `renderAsync` with adding more clues.

Reply:
Surprisingly, the tests passed in the experiment PR but they consistently failing locally. I have 10-15 failing tests for a single run.

Reply:
Applied the same `describe.each` multiplier, ran tests a few thousand times locally as well, all passes for me.

Do you consider your local issues merge blocking?

Reply:
Alright, it is not a blocker for the pr then. I use the same branch and run npm install - the tests do fail consistently for me still.

Reply:
Did you try [this](https://github.com/cloudscape-design/components/pull/2677) version, right?

I notices you removed lines like `expect(findAllBreadcrumbsInstances()).toHaveLength(1);`. All failures I receive locally are due to these.

Reply:
The latest version with `expect(findAllBreadcrumbsInstances()).toHaveLength(1)` under waitFor does work for me locally, too.